tests/test_inclined_exponential.py

```diff
@@ -30,173 +35,340 @@
     sys.path.append(os.path.abspath(os.path.join(path, "..")))
     import galsim
 
+save_profiles = True
+
 # set up any necessary info for tests
 # Note that changes here should match changes to test image files
 image_dir = './inclined_exponential_images'
 
 # Values here are strings, so the filenames will be sure to work (without truncating zeros)
-fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
-image_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
-image_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
-image_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
-image_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# InclinedExponential test cases, which we'll also test InclinedSersic with
+both_fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
+both_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
+both_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
+both_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
+both_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# InclinedSersic-only test cases
+sersic_fluxes = ("1.0", "2.0", "1.5", "1.1", "10.0", "1.0e6", "1.0e-6", "2.3e4",)
+sersic_ns = ("1.0", "1.5", "2.0", "2.5", "1.0", "1.5", "2.0", "2.5",)
+sersic_inc_angles = ("0.0", "0.0", "0.0", "0.0", "0.1", "0.2", "0.3", "0.5",)
```

Comment:
It's ok to have the regression tests only cover the range where it is reasonable to run Lance's code (and where it doesn't take forever).  The problem is really that this same list is also used for the sanity tests, which cover a bit more in terms of the correctness of the functionality.  I propose just adding more elements to this set that are only run for the sanity tests, not the regression tests.